Cassandra/ORM/SchemaManager.php

```diff
@@ -64,12 +64,14 @@ public function createTable($name, $fields, $primaryKeyFields = [], $tableOption
 
         if (!empty($tableOptions)) {
             $tableOptionsParamCQL = [];
-            if (isset($tableOptions['compactStorage']) && false !== $tableOptions['compactStorage']) {
-                $tableOptionsParamCQL[] = 'COMPACT STORAGE';
-            }
-
-            if (isset($tableOptions['clusteringOrder']) && null !== $tableOptions['clusteringOrder']) {
-                $tableOptionsParamCQL[] = sprintf('CLUSTERING ORDER BY (%s)', $tableOptions['clusteringOrder']);
+            foreach ($tableOptions as $optionName => $optionValue) {
+                if ('compactStorage' === $optionName && false !== $optionValue) {
+                    $tableOptionsParamCQL[] = 'COMPACT STORAGE';
+                } elseif ('clusteringOrder' === $optionName && null !== $optionValue) {
+                    $tableOptionsParamCQL[] = sprintf('CLUSTERING ORDER BY (%s)', $optionValue);
+                } elseif (!\in_array($optionName, ['compactStorage', 'clusteringOrder'])) {
```

Comment:
if the code reach this statement, the `$optionName` shouldn't be `compactStorage` or `clusteringOrder` right ? so i think we can remove the checking and just use `else` instead of `elseif`. wdyt ?

Reply:
Hello @Hendra-Huang,
For example, if $optionName === 'clusteringOrder' and $optionValue is null, then by using just an else, I could get an CQL with the table option : `WITH clusteringOrder = ""`
I think I must keep the elseif (which I don't like so much too but which is required for me).

Reply:
oh okay got it.. thanks